sdk-api-src/content/dxgi1_5/nf-dxgi1_5-idxgioutput5-duplicateoutput1.md

```diff
@@ -62,9 +62,9 @@ A pointer to the Direct3D device interface that you can use to process the deskt
 
 ### -param Flags
 
-Type: <b>UINT</b>
+Type: <b>DXGI_OUTDUPL_FLAG</b>
```

Comment:
I don't think this `Type:` field can change, but the description looks good to me.

Reply:
Description was randomly thought up, unsure if you want to go with the "surface" wording.